### DISCUSSION TITLE
the map ends due to "failure" with the Relax mod enabled

### DISCUSSION THREAD

**Discussion by Baldaeb228** [ppy/osu#26100] [Q&A] [answered]
the map in the game ends due to “failure” with the Relax mod enabled, literally yesterday this did not happen, but now the map becomes a failure if the HP indicator drops to zero. Map has never completed before with Relax enabled

**Comment by cihe13375** [comment]
[Yes they added this recently](https://github.com/ppy/osu/pull/25917)

**Reply by Baldaeb228** [reply]
Is there any way to disable this option? Very disturbing

**Reply by BlauFx** [reply, marked as answer]
Enable no fail mod

**Reply by Baldaeb228** [reply]
no fail reduces the modifier by 0.5, then it’s easier to set difficulty adjust and set more HP there, at the same time changing the game as you want. It seems to me that those who play on relaxation for points will not like that they will receive half as much

**Reply by Baldaeb228** [reply]
I think it would be more logical to add 2 Relax, one with Fail, the other without, or change this option in fine tuning
